core.js

```diff
@@ -253,6 +259,46 @@ function create_patch_request(ch, pull) {
 }
 
 /*
+Replace individual constants in formulae with actual values
+and supply a result.
+
+If pull argument is missing, test arithmetic expression using a default value
+and try to evaluate expression. Returns null if failed.
+
+IL - Inserted lines
+RL - Removed lines
+FC - Files changed
+R  - Rating
+*/
+exports.eval_formulae = function(formulae, pull) {
+
+  // Store final result
+  score = null
```

Comment:
You don't need this, because it evaluates the formulae, and if there is an error its set to null. This will be overwritten either way.